frontend/src/App.jsx

```diff
@@ -1,12 +1,32 @@
 import Card from "@components/Cards/Card/Index";
-import CardBack from "@components/Cards/CardsBack/Index";
-
 import Logo from "@components/Header/Logo/Logo";
 import Titre from "@components/Header/Titre/Titre";
 import SearchButton from "@components/SearchButton";
+import axios from "axios";
+import React, { useEffect } from "react";
+
 import "./App.css";
 
 function App() {
+  const [movies, setMovies] = React.useState([]);
+  const getMovies = () => {
+    axios
+      .get(
+        "https://api.themoviedb.org/3/movie/popular?api_key=f352c00b6a6dc4de283669f3f964df1a&language=en-US&page=1"
+      )
+      .then((response) => response.data)
+      .then((data) => {
+        setMovies(data.results);
+      });
+  };
+  useEffect(() => {
+    getMovies();
+  }, []);
+
+  const filter18 = movies.filter((movie) => {
+    return !movie.genre_ids.includes(27);
+  });
```

Comment:
Doing code here is not a good idea, it should be in the useEffect 
See previous comment

Reply:
See previous :-D